packages/lib-react-components/src/Markdownz/Markdownz.spec.js

```diff
@@ -87,7 +89,7 @@ describe('<Markdownz />', function () {
         const baseURL = ''
         const expectedReturnValue = `${baseURL}/talk/search?query=%23${resource}`
         wrapper.instance().buildResourceURL(resource, '#')
-        expect(buildResourceURLSpy).to.have.been.calledOnce()
+        expect(buildResourceURLSpy.withArgs(resource, '#')).to.have.been.calledOnce()
```

Comment:
I don't understand this test. The preceding line called `buildResourceURL(resource, '#')` so what is this expectation checking?